SubLink/Program.cs

```diff
@@ -60,7 +60,10 @@ internal partial class Program {
     "Webhook": ""
   },
   "SubLink": {
-    "Discriminator": {discriminator}
+    "Discriminator": {discriminator},
+    "OscIPAddress": "127.0.0.1",
+    "OscPort": 9000,
+    "ScriptName": "SubLink.cs"
```

Comment:
If they have non-null/non-empty defaults I feel like they shouldn't be in the default settings file, but documented instead.

Also, I feel like the OSC settings are not inherently SubLink settings but I might be splitting hairs a bit too much.